domain/camera/src/main/java/com/google/jetpackcamera/domain/camera/CameraXCameraUseCase.kt

```diff
@@ -541,6 +566,10 @@ constructor(
     private val _zoomScale = MutableStateFlow(1f)
     override fun getZoomScale(): StateFlow<Float> = _zoomScale.asStateFlow()
 
+    private val _lowLightBoostActiveStatus = MutableStateFlow(true)
```

Comment:
This would mean that low light is active?
a) maybe the enum approach I suggested above might be better?
b) maybe starting as `inactive` might be better?
c) if I got the active / inactive wrong then it might be better to type it to an enum to avoid confusion